### PR TITLE
Documentation on release procedure

### PR DESCRIPTION
**Pull request by jh-RLI**
- Documentation of the deployment pipeline
- Basic-Steps for deploy and release (publishing a new release)

### CODE REVIEW

File: RELEASE_PROCEDURE.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+1. Switch to the now-updated master branch: `git checkout master` and `git pull upstream master`
+1. Tag the release number: `git tag v<release version>`, e.g., `git tag v1.2.0`
+   - `versioneer` automatically updates the version number based on the tag
+   - this is now the official tagged commit
+1. Push the tag upstream: `git push upstream --tags`
+1. Make a new release on Github
+   - make sure that you choose the tag name defined above
+   - copy the release summary from changelog into the description box
+1. Add a new line "# Next Release" at the top of `RELEASE_NOTES.md` and commit to `master`
+1. Announce it on our mailing list: https://groups.io/g/pyam
```
Comment (MGlauer) [inline RELEASE_PROCEDURE.md:47]: Is this the wrong link?

File: RELEASE_PROCEDURE.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+  1. Once successful, delete the tag, and merge the candidate PR into `master` on Github
+1. Switch to the now-updated master branch: `git checkout master` and `git pull upstream master`
+1. Tag the release number: `git tag v<release version>`, e.g., `git tag v1.2.0`
+   - `versioneer` automatically updates the version number based on the tag
+   - this is now the official tagged commit
+1. Push the tag upstream: `git push upstream --tags`
+1. Make a new release on Github
+   - make sure that you choose the tag name defined above
+   - copy the release summary from changelog into the description box
+1. Add a new line "# Next Release" at the top of `RELEASE_NOTES.md` and commit to `master`
```
Comment (MGlauer) [inline RELEASE_PROCEDURE.md:46]: What is the difference between `RELEASE_NOTES.md` and the versions above? Do we need both?

File: RELEASE_PROCEDURE.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+1. Announce it on our mailing list: https://groups.io/g/pyam
+   - again, copy the rendered HTML from the Github release directly in the email
+
+1. Deploy on webserver: https://docs.djangoproject.com/en/3.0/howto/deployment/
```
Comment (MGlauer) [inline RELEASE_PROCEDURE.md:50]:
```suggestion
1. As stated in the [django deployment documentation](https://docs.djangoproject.com/en/3.0/howto/deployment/), every django app has to be served by an external web server. Please make sure to follow the security advise given in the deployment documentation, if you want to host your own version of the Open Energy Platform.
```

File: RELEASE_PROCEDURE.md (hunk 1 above)
Comment (jh-RLI) [reply RELEASE_PROCEDURE.md:47]: do we have a mailing list I´m not aware of? I just left it there as an example.

File: RELEASE_PROCEDURE.md (hunk 2 above)
Comment (jh-RLI) [reply RELEASE_PROCEDURE.md:46]: I will rename it to "RELEASE_PROCEDURE.md" :)

**Comment by jh-RLI** [comment]
@MGlauer is this about it? Is something major missing?

File: RELEASE_PROCEDURE.md (hunk 1 above)
Comment (jh-RLI) [reply RELEASE_PROCEDURE.md]: @MGlauer ping :)

**Pull request merged**